Scoop/Controller/Ride Posting/PostRideTripDetailsViewController.swift

```diff
@@ -55,9 +55,12 @@ class PostRideTripDetailsViewController: OnboardingViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .white
-        navigationItem.title = "Trip Details"
         
-        let nextAction = UIAction { _ in
+        nextAction = UIAction { _ in
+            guard let navCtrl = self.navigationController else {
+                return
+            }
```

Comment:
Put this in one line `guard let navCtrl = self.navigationController else { return }`